tests/test_runtime_build.py

```diff
@@ -22,7 +22,7 @@ def test_placeholders():
     f = hcl.build(s, target)
     print(f)
 
-def test_debug_mode():
+def test_sdaccel_debug():
```

Comment:
I would suggest to move all the test cases of debug mode inside a single test function, but you do not have to that now.

Reply:
Okay, I'll fix it.